### ISSUE TITLE
Issue with including code files via `literalinclude`

### ISSUE THREAD

**Issue by mmcky** [executablebooks/sphinx-tomyst#92]
There is an issue with translation of `literalinclude` to `myst`

```md
.. literalinclude:: /_static/lecture_specific/optgrowth_fast/ogm.py
```

includes the contents of the file in the `myst` output. 

This helps with not requiring to copy over assets when translating to `myst` but isn't a particularly friendly approach to translation. 

We could:

- ~~track `literalinclude` files + paths and copy them across. Then we can include `{literalinclude}` directives~~
- include the contents of the files in the output `myst` but add support for `code-cell` for `myst_nb` target option.

**Comment by mmcky** [comment]
The `_static` folder is currently copied to bring across `figures` -- so if we assume there are no externally referenced files (which I think is reasonable assumption for now) then we should

- [ ] include `literalinclude` direct in the `myst` rather than include the content.

**Comment by mmcky** [comment]
https://github.com/executablebooks/MyST-NB/issues/274

**Comment by mmcky** [comment]
Perhaps the best solution will be as suggested in https://github.com/executablebooks/MyST-NB/issues/274 and translate `literalinclude` items to `code-cell` that is read in from a file. 

**Updated Decision:**

I think `sphinx-tomyst` should actually be updated to add `literalinclude` directive and a file reference (as that is a true represnetation of the `rst`) and any adaptation required to bring in `executability` to the `jupyter-book` workflow is an update to the source markup

**Comment by mmcky** [comment]
fixed by PR #93
